cves/kernel/CVE-2016-2782.yml

```diff
@@ -89,9 +89,7 @@ fixes:
 - commit:
```

Comment:
Add the commit it was introduced, see [this](https://ubuntu.com/security/CVE-2016-2782)

Reply:
I think this commit is already on the report. Line 106 has the commit that introduced the issue.